### PR TITLE
fixing typos

### PR DESCRIPTION
**Pull request by Budalebah**
fixing typos

### CODE REVIEW

File: Introduction to zkVerify Blockchain/2. Introducing and Architecting zkVerify/4. Introducing zkVerify.md
Comment (snehaweb3kid) [inline Introduction to zkVerify Blockchain/2. Introducing and Architecting zkVerify/4. Introducing zkVerify.md:1]: web3 does not need to be changed to Web3

**Review by snehaweb3kid — APPROVED** [review]
web3 does not need to be changed to Web3, rest changes are fine.

**Comment by snehaweb3kid** [comment]
no major changes.

**Review by snehaweb3kid — COMMENTED** [review]
Approved

**Review by snehaweb3kid — APPROVED** [review]

**Comment by Budalebah** [comment]
Thanks

**Pull request merged**